### DISCUSSION TITLE
Unable to log in on Windows start up or after screen lock

### DISCUSSION THREAD

**Discussion by rensimrs** [rustdesk/rustdesk#10155] [General]
### Bug Description

After screen lock it is not possible anymore to input the password to unlock the screen. The mouse and keyboard look like they get detached from the remote device. This detaching also happens when some confirmation popup window appear to confirm permissions of a higher degree

### How to Reproduce

Lock the screen or reboot the pc after the latest windows updates (windows 11 and 10)

### Expected Behavior

it used to allow normally the login at windows logon

### Operating system(s) on local (controlling) side and remote (controlled) side

windows 10 and 11

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.2.2, 1.3.3

### Screenshots

![photo_2024-12-03_11-23-48](https://github.com/user-attachments/assets/f04c6ced-db7d-4b2b-a14d-87c581ed2376)

**Comment by hrpv** [comment]
### Windows 11 24H2 KB5086672 (Build 26200.8117) - No keyboard input at login screen with RustDesk 1.4.6
System: Windows 11 Pro 24H2 Build 26200.8117
#### Update: KB5086672 (March 2026) - Cumulative Update
RustDesk Version: 1.4.6 (latest version)
Installation mode: Installed as service
Problem
After installing KB5086672 (March 2026 Cumulative Update), no keyboard or mouse input is possible at the Windows login screen via RustDesk.
Symptoms
RustDesk connection establishes successfully, screen is transmitted
Mouse not displayed
Keyboard completely blocked at login screen (no input possible)
Problem occurred immediately after KB5086672 installation (restart required)
Mouse and Physical keyboard on-site works normally
Even with Windows AutoLogon enabled no mouse or keyboard input

Important observation
#### AnyDesk Version 9 continues to work, indicating the problem is RustDesk-specific and not caused by general Windows security policies. AnyDesk appears to use a different architecture for accessing the secure desktop (Winlogon).
##### Current status
RustDesk 1.4.6 uninstalled. Waiting for fix. AnyDesk Version 9 as temporary alternative in use.
##### Possible connection
KB5086672 is officially a "fix update" for previous update issues, but apparently causes new regressions with remote desktop connections. Parallel problems were reported with Windows RDP after similar updates (KB5051987, KB5050094), which were resolved by disabling "Network Detection" in Group Policies. May be relevant for troubleshooting.

**Update:** 
**uninstall rustdesk and cleaning really every rustdesk traces on the system** in
Program files, registry, Firewall, drivers and reinstall brought the system back to work
Simple uninstall and reinstall didnt work
